Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -302,7 +316,8 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
     }
 
     fileprivate func animateSearchEnginesWithKeyboard(_ keyboardState: KeyboardState) {
-        layoutSearchEngineScrollView()
+        // Ecosia: remove search customization
```

Comment:
Just a thought and no need to change anything now, but probably it would make things easier in the future if we stick to always comment out something like
```
/* Ecosia: ...
*/
```
Instead of the `//`. It least it might help while searching for all places where we have commented things.

Reply:
yes true - it was a single line here so I opted for `//` happy to strive for consistency 👍